src/javascripts/jquery.tipsy.js

```diff
@@ -118,6 +118,21 @@
                 }
             }
         },
+
+        destroy: function(){
+            this.$element.removeData('tipsy');
```

Comment:
Realised that we probably only wants to remove the "tipsy" data so that we play nicely with other plugins.